pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -544,7 +562,7 @@ func (configgen *ConfigGeneratorImpl) buildInboundClusters(env *model.Environmen
 				ManagementClusterHostname, port.Port)
 			localityLbEndpoints := buildInboundLocalityLbEndpoints(actualLocalHost, port.Port)
 			mgmtCluster := buildDefaultCluster(env, clusterName, apiv2.Cluster_STATIC, localityLbEndpoints,
-				model.TrafficDirectionInbound, proxy, nil)
+				model.TrafficDirectionInbound, proxy, nil, true) // TODO GregHanson management ports always mTLSReady?
```

Comment:
what is management server/port? pilot? isn't that part of bootstrap static config then?